### DISCUSSION TITLE
iOS Home indicator

### DISCUSSION THREAD

**Discussion by Arjan-Zuidema** [zoontek/react-native-edge-to-edge#47] [Ideas]
Hey @zoontek,

First thanks for this package!

Would it fit in this package to be able to set the iOS Home indicator to preferred hidden? I think it would, as it's also some kind of SystemBar :)

**Comment by zoontek** [comment]
@Arjan-Zuidema Hey! That's a good idea, but for the moment the package does not have an iOS native part, so this will come later 🙂
